src/main/java/org/neo4j/junit/jupiter/causal_cluster/CausalClusterExtension.java

```diff
@@ -35,7 +38,9 @@
 import org.junit.jupiter.api.extension.ExtensionContext;
 import org.junit.jupiter.api.extension.TestInstances;
 import org.junit.platform.commons.support.AnnotationSupport;
-import org.junit.platform.commons.support.HierarchyTraversalMode;
+import org.testcontainers.containers.Neo4jContainer;
+
+import static org.junit.platform.commons.support.HierarchyTraversalMode.TOP_DOWN;
```

Comment:
I wonder if that goes through my check style config… :hmm:

Reply:
it doesn't.